java/src/main/java/com/datasqrl/ai/models/openai/ChatModel.java

```diff
@@ -1,22 +1,23 @@
 package com.datasqrl.ai.models.openai;
 
+import com.datasqrl.ai.models.GenericLanguageModel;
 import com.knuddels.jtokkit.api.ModelType;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 
 @AllArgsConstructor
 @Getter
-public enum ChatModel {
+public enum ChatModel implements GenericLanguageModel {
```

Comment:
I have a compilation problem here. Can you pls help @mbroecheler ?
`java: com.datasqrl.ai.models.groq.ChatModel is not abstract and does not override abstract method getCompletionLength() in com.datasqrl.ai.models.GenericLanguageModel`

Reply:
Doesn't the @Getter automatically generate this method?

Reply:
OK, I fixed this issue myself. Still, please take a look at the Enum inheritance and let me know if it could be made prettier

Reply:
Yes, using an interface is the best solution.